config_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+// in order to test command line arguments and environment variables
+// you will need to append the flags to the os.Args variable like so
+// os.Args = append(os.Args, "--altdnsnames=\"hostname1,hostname2\"")
+// For environment variables you can use the
+// os.Setenv("DCRD_ALT_DNSNAMES", "hostname1,hostname2") to set the variable
+// before loadConfig() is called
+// These args and env variables will then get parsed by loadConfig()
+
+func setup() {
+	// Temp config file is used to ensure there are no external influences
+	// from previously set env variables or default config files.
+	file, _ := ioutil.TempFile("", "dcrd_test_file.cfg")
+	defer os.Remove(file.Name())
+	// Parse the -test.* flags before removing them from the command line
```

Comment:
There should be a blank line before this for consistency with the rest of code base.